google/cloud/storage/bucket_metadata_test.cc

```diff
@@ -641,8 +646,22 @@ TEST(BucketMetadataTest, SetIamConfigurationUBLA) {
       << "\n  actual=" << copy.iam_configuration() << "\n";
 }
 
+/// @test Verify we can change the IAM Configuration in BucketMetadata.
+TEST(BucketMetadataTest, SetIamConfigurationPAP) {
+  auto expected = CreateBucketMetadataForTest();
+  auto copy = expected;
+  BucketIamConfiguration new_configuration;
+  new_configuration.public_access_prevention = "enforced";
+  copy.set_iam_configuration(new_configuration);
+  ASSERT_TRUE(copy.has_iam_configuration());
+  EXPECT_EQ(new_configuration, copy.iam_configuration());
+  EXPECT_NE(expected, copy)
+      << "expected = " << expected.iam_configuration()
+      << "\n  actual=" << copy.iam_configuration() << "\n";
```

Comment:
Perhaps a space before the `=`, like in `"expected ="`?

Reply:
Fixed here and elsewhere.